Workbooks/Azure SQL VM/SQL Assessment/SQL Assessment.workbook

```diff
@@ -0,0 +1,661 @@
+            "isGlobal": true,
+            "value": "",
+            "isHiddenWhenLocked": true
+          },
+          {
+            "id": "c10f05b2-6ac2-4392-9ad0-f4eadf4ae632",
+            "version": "KqlParameterItem/1.0",
+            "name": "typeSystem",
+            "type": 1,
+            "value": "[\"CPU\", \"IO\",\"Storage\"]",
+            "isHiddenWhenLocked": true
+          },
+          {
+            "id": "6681e8b9-4cfc-4868-9875-f0b8138f15cf",
+            "version": "KqlParameterItem/1.0",
+            "name": "typeAdmins",
+            "type": 1,
+            "value": "[\"TraceFlag\",\"Backup\",\"DBCC\", \"DBConfiguration\",\"SystemHealth\",\"Traces\",\"DBFileConfiguration\",\"Configuration\",\"Replication\",\"Agent\",\"Security\",\"DataIntegrity\",\"MaxDOP\",\"PageFile\"]",
+            "isHiddenWhenLocked": true
+          },
+          {
+            "id": "e30520e4-f856-4611-8783-2fe994993233",
+            "version": "KqlParameterItem/1.0",
+            "name": "typeDevs",
+            "type": 1,
+            "value": "[\"UpdateIssues\",\"Index\",\"Naming\",\"Deprecated\", \"masterDB\",\"QueryOptimizer\",\"QueryStore\"]",
+            "isHiddenWhenLocked": true
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces"
+      },
+      "name": "Parameters Group"
+    },
+    {
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "items": [
+          {
+            "type": 12,
+            "content": {
+              "version": "NotebookGroup/1.0",
+              "groupType": "editable",
+              "items": [
+                {
+                  "type": 1,
+                  "content": {
+                    "json": "**<span style=\"font-size: 2em\">SQL Assessment Results</span>**"
+                  },
+                  "customWidth": "18",
+                  "name": "text - 1"
+                },
+                {
+                  "type": 9,
+                  "content": {
+                    "version": "KqlParameterItem/1.0",
+                    "parameters": [
+                      {
+                        "id": "ca3f7f64-12c1-49c9-b5bc-b29ba33e4d45",
+                        "version": "KqlParameterItem/1.0",
+                        "name": "AsmtId",
+                        "label": " collected at",
+                        "type": 2,
+                        "isRequired": true,
+                        "query": "SqlAsmtLogs_CL\r\n| where TimeGenerated > ago(180d) and _ResourceId =~ {VmResourceId:value}\r\n| extend asmt = parse_csv(RawData)\r\n| project AsmtId = tostring(asmt[1]), TimeGenerated\r\n| summarize AsmtTime = max(TimeGenerated) by AsmtId\r\n| order by AsmtTime desc\r\n| extend rn=row_number()\r\n| project value = AsmtId, label = strcat(format_datetime(AsmtTime, 'yyyy-MM-dd hh:mm tt'), ' UTC'), selected = rn == 1",
```

Comment:
NIT: Switch to SqlAssessment_CL,  asmt wont be clear to advanced users reviewing Log Analytics configuration

Reply:
are you suggesting replacing all SqlAsmtLogs_CL with SqlAssessment_CL in the workbook? I don't think SqlAssessment_CL will be recognized, Qingquan just tried and it doesn't work. What else should we modify to support SqlAssessment_CL?

Reply:
The custom log definition would also need to be updated to `SqlAssessment` (_CL is automatically added by LA)

![image](https://user-images.githubusercontent.com/52937801/123332127-f9069900-d4f4-11eb-8fa2-75ca715ec7a4.png)

Reply:
I've replaced all SqlAsmtLogs_CL with SqlAssessment_CL, should be good now